library/std/src/fs.rs

```diff
@@ -31,6 +31,11 @@ use crate::time::SystemTime;
 /// on closing are ignored by the implementation of `Drop`.  Use the method
 /// [`sync_all`] if these errors must be manually handled.
 ///
+/// `File` objects do not buffer reads and writes by default. For efficiency,
+/// consider wrapping the file in a [`BufReader`] or [`BufWriter`] when performing
+/// multiple [`read`] or [`write`] calls, unless unbuffered reads and writes
```

Comment:
It should say *small* reads and writes. You won't get any speedup if you're performing multiple large operations that exceed the buffer size.